ci/run.sh

```diff
@@ -24,16 +24,26 @@ if [ "${NO_STD:-}" = "1" ]; then
     echo "nothing to do for no_std"
 else
     run="cargo test --manifest-path testcrate/Cargo.toml --no-fail-fast --target $target"
-    $run
+
+    if [[ ! "$target" =~ sbf* && ! "$target" =~ sbpf-* && ! "$target" =~ sbpfv3- ]]; then
+      # Not using release mode causes a stack overflow in SBPFv0
+      # There is a bug in SBPFv3 whereby we were not adding returns to -O0 code
```

Comment:
The fix for SBPFv3 is in https://github.com/anza-xyz/llvm-project/pull/150, and it requires another platform tools release. To avoid back-and-forths between repos, I am only going to remove the restriction here once I need to touch on this repository again.